README.md

```diff
@@ -1,58 +1,78 @@
-# GriefPrevention - Minecraft anti-griefing plugin
+<img align="right" src="https://media.forgecdn.net/avatars/73/181/636163527796328705.png" height="200" width="200">
 
-Stop responding to grief and prevent it instead. Grief Prevention will solve
-your grief problems without a roster of trained administrators, without 10
-different anti-grief plugins, and without disabling any standard game features.
-Because Grief Prevention teaches players for you, you won't have to publish a
-training manual or tutorial for players, or add explanatory signs to your world.
+# GriefPrevention - Minecraft anti-griefing plugin [![Build Status](https://github.com/TeamWuffy/GriefPrevention/workflows/Build/badge.svg)](https://github.com/TeamWuffy/GriefPrevention/actions?query=workflow%3ABuild) [![Release Status](https://github.com/TeamWuffy/GriefPrevention/workflows/Releases/badge.svg)](https://github.com/TeamWuffy/GriefPrevention/releases/latest)
```

Comment:
Appreciate the PR but this diff is way too big and is apparently customized for your fork. Please consider breaking up this PR into smaller PRs that each address a single issue.

Reply:
okay